@shared/api/internal.ts

```diff
@@ -193,31 +199,41 @@ export const getAccountDetails = async (
     networkPassphrase: NETWORK_PASSPHRASE,
   });
 
-  let payments = null;
   let balances = null;
   let isFunded = null;
+  let operations = [] as Array<HorizonOperation>;
 
   try {
     ({ balances } = await dataProvider.fetchAccountDetails());
-    const transactionData = await dataProvider.fetchPayments({
-      limit: TRANSACTIONS_LIMIT,
-    });
-    payments = transactionData?.records || null;
+    isFunded = true;
   } catch (e) {
     console.error(e);
+    return {
+      balances,
+      isFunded: false,
+      operations,
+    };
   }
 
   try {
-    isFunded = await dataProvider.isAccountFunded();
+    const server = new StellarSdk.Server(NETWORK_URL);
+    const operationsData = await server
```

Comment:
If we don't need non-payment operations, we can use `stellar/wallet-sdk`s `fetchPayments()` method, which simplifies this call a bit. [Example ](https://github.com/stellar/account-viewer-v2/blob/master/src/ducks/txHistory.ts#L28:L41)in AV.

Reply:
Ah, unfortunately, we do need all types of operations